### PR TITLE
LRDOCS-8675 Search Custom Facet Improvements Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-8675

@rbohl
@lipusz

- LRDOCS-8675 further promote the use of the Field Mappings tab
- LRDOCS-8675 talk about multi-fields and .raw
- LRDOCS-8675 adding custom fields and finding raw fields
- LRDOCS-8675 document the ddm nested fields change
- LRDOCS-8675 refinements
- LRDOCS-8675 sme review: create index reference article, document legacy setting, remove notes
- LRDOCS-8675 link to new index reference article
- LRDOCS-8675 Index Reference > Elasticsearch Indexes Reference
- LRDOCS-8675 Wordsmithing
- LRDOCS-8675 309 Wordsmithing.

### CODE REVIEW

File: docs/dxp/latest/en/using-search/search-administration-and-tuning/index-reference.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Elasticsearch Indexes Reference
```
Comment (jrhoun) [inline docs/dxp/latest/en/using-search/search-administration-and-tuning/index-reference.md:1]: @rbohl it _seems_ like this article is the same as `elasticsearch-indexes-reference.md`? Is that intended?

Comment (jrhoun) [reply docs/dxp/latest/en/using-search/search-administration-and-tuning/index-reference.md:1]: Going to remove this as I'm pretty sure it's unintended and it isn't added to the nav. Flag me down if this is incorrect.

**Comment by jrhoun** [comment]
Merged. Thanks. See my specific changes here: https://github.com/liferay/liferay-learn/commit/8373c7dd92d5a8f337e230022ba5f17bf697084c

**Comment by rbohl** [comment]
Yeah that looks like a duplicate article, cruft from a rename I did most likely. Thanks for killing it!

**Pull request closed without merging**